tests/lib/bin/mountinfo-tool

```diff
@@ -421,7 +556,9 @@ Known attributes, applicable for both filtering and display.
     """,
         formatter_class=RawTextHelpFormatter,
     )
+    parser.register("action", "self-test", _SelfTestAction)
     parser.add_argument("-v", "--version", action="version", version="1.0")
+    parser.add_argument("--self-test", action="self-test")
```

Comment:
self-test for me means usually something else than this, is more about a tools that have state etc and checking for consistency of that state.  I would call the option "--run-code-tests".

Reply:
I took this from “bzr” which did exactly that. I can rename it to “—unit-tests” to be explicit

Reply:
slight preference for the longer "--run-unit-tests"

Reply:
Done